### ISSUE TITLE
Python 3 conda build broken for DLS eiger images

### ISSUE THREAD

**Issue by rjgildea** [dials/dials#1188]
```
$ module load dials/now
$ dials.import /dls/i04/data/2020/cm26459-1/20200220/TestInsulin/ins_1/ins_1_1_master.h5
DIALS (2018) Acta Cryst. D74, 85-97. https://doi.org/10.1107/S2059798317017235
DIALS 2.dev.1438-g3e404549a
Traceback (most recent call last):
  File "/dls/science/groups/scisoft/DIALS/CD/now/build_dials/build/../modules/dials/command_line/dials_import.py", line 921, in <module>
    script.run()
  File "/dls/science/groups/scisoft/DIALS/CD/now/build_dials/build/../modules/dials/command_line/dials_import.py", line 752, in run
    params, options = self.parser.parse_args(args=args, show_diff_phil=False)
  File "/dls/science/groups/scisoft/DIALS/CD/now/build_dials/modules/dials/util/options.py", line 852, in parse_args
    quick_parse=quick_parse,
  File "/dls/science/groups/scisoft/DIALS/CD/now/build_dials/modules/dials/util/options.py", line 569, in parse_args
    load_models=load_models,
  File "/dls/science/groups/scisoft/DIALS/CD/now/build_dials/modules/dials/util/options.py", line 235, in __init__
    load_models,
  File "/dls/science/groups/scisoft/DIALS/CD/now/build_dials/modules/dials/util/options.py", line 308, in try_read_experiments_from_images
    load_models=load_models,
  File "/dls/science/groups/scisoft/DIALS/CD/now/build_dials/modules/dxtbx/model/experiment_list.py", line 536, in from_filenames
    format_kwargs=format_kwargs,
  File "/dls/science/groups/scisoft/DIALS/CD/now/build_dials/modules/dxtbx/datablock.py", line 1529, in from_filenames
    format_kwargs=format_kwargs,
  File "/dls/science/groups/scisoft/DIALS/CD/now/build_dials/modules/dxtbx/datablock.py", line 948, in __init__
    format_class = find_format.find_format(filename)
  File "/dls/science/groups/scisoft/DIALS/CD/now/build_dials/modules/dxtbx/datablock.py", line 329, in find_format
    self._format_class = get_format_class_for_file(filename)
  File "/dls/science/groups/scisoft/DIALS/CD/now/build_dials/modules/dxtbx/format/Registry.py", line 106, in get_format_class_for_file
    format_class = get_format_class_for(format)
  File "/dls/science/groups/scisoft/DIALS/CD/now/build_dials/modules/dxtbx/format/Registry.py", line 17, in get_format_class_for
    return get_format_class_index()[format_class_name][0]()
  File "/dls/science/groups/scisoft/DIALS/CD/now/build_dials/conda_base/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2446, in load
    return self.resolve()
  File "/dls/science/groups/scisoft/DIALS/CD/now/build_dials/conda_base/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2452, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/dls/science/groups/scisoft/DIALS/CD/now/build_dials/modules/dlstbx/format/FormatEiger0MQDump.py", line 7, in <module>
    import bitshuffle
  File "/dls/science/groups/scisoft/DIALS/CD/now/build_dials/conda_base/lib/python3.6/site-packages/bitshuffle/__init__.py", line 20, in <module>
    from bitshuffle.ext import (__version__, bitshuffle, bitunshuffle, using_NEON, using_SSE2,
  File "__init__.pxd", line 918, in init bitshuffle.ext
ValueError: Please report this error to dials-support@lists.sourceforge.net: numpy.ufunc size changed, may indicate binary incompatibility. Expected 216 from C header, got 192 from PyObject
```

**Comment by Anthchirp** [comment]
fixed via #1193 in 80b50b0f1ab6d3703a18199721abd2d3c35140bd

**Comment by Anthchirp** [comment]
to fix an existing installation:
`libtbx.conda install -p $(libtbx.show_build_path)/../conda_base "numpy>=1.16,<1.17"`
